src/error.rs

```diff
@@ -2,12 +2,13 @@
 //!
 
 use alloc::string::String;
+#[cfg(not(feature = "std"))]
+use core::error;
```

Comment:
I believe you can remove the cfg above this, and the entire cfg and import below by simply doing std::io::Error